.github/workflows/dev-check.yml

```
@@ -76,31 +126,34 @@ jobs:
-          
-          branch: ${{ github.ref_name }}
-          author: ${{ github.actor }}
-          track: Internal Testing
-          status: Uploaded to Play Store & GitHub"
+
+Branch: ${{ github.ref_name }}
+Author: ${{ github.actor }}
+Track: Internal Testing
+Status: Uploaded to Play Store & GitHub
+
+"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)